### ISSUE TITLE
Sequence for non-variable sites

### ISSUE THREAD

**Issue by rneher** [nextstrain/auspice#616]
I think we should have sequence info for sites that are not variable. The current behavior is not good:

https://nextstrain.org/zika?c=gt-NS1_347

Saving sequence in the json would not be a big overhead for viruses (a few kb). For bacteria, it's more of an issue (about a 2MB assuming gzip). 
It would also clean up code here
https://github.com/nextstrain/auspice/blob/master/src/util/setGenotype.js#L22
since we no longer have to do infer the ancestral state from mutations but could simply look up the ancestral state in a well-defined location.

**Comment by jameshadfield** [comment]
I tend to agree - especially with Emma's improvements of the entropy panel. I'd propose including this as a property on the meta/unified JSON. If this is too slow for bacteria, we can turn this into a separate JSON and fetch it in the background.

**Comment by rneher** [comment]
yes, we could have smth like 
```
ancestral_sequence:{'nuc':'ACTTG...', 'NP1':'MTHGYF...'}
```
in the `meta.json`. This would also allow rendering the actual sequence in the entropy panel once one has zoom in enough to see letters.

**Comment by trvrb** [comment]
If we include this (which seems like a fine idea), my suggested unified JSON structure would be:
```
meta: {},
tree: {},
sequence: {}
```
I don't think this be part of `meta`. This is mainly because we might pull this data out into its own JSON (`tb_sequence.json`) for loading bacterial datasets. And this is partially because, like `tree`, `sequence` feels overly "weighty" / data-like to be just another element in `meta`.

**Comment by rneher** [comment]
This would suggest that we export it as a separate `sequence.json` and the web-server combines it into a single one if necessary.

**Comment by jameshadfield** [comment]
My understanding of the proposed solution (after various phone calls) is:
* sequence information will be in a separate json file
* a flag in the meta json (or unified json) will indicate that this is present
* auspice will subsequently fetch this json

**Comment by jameshadfield** [comment]
Current status of this issue:

Augur v6 will produce a `<name>_root_sequence.json` if requested, as per https://github.com/nextstrain/augur/pull/417. Auspice should attempt to fetch this file and use it to allow colouring by non-variable positions in the genome (via the dropdowns or by defining it in the URL). The fetch should be made in the same fashion as the one for tip-frequencies JSONs (https://nextstrain.github.io/auspice/server/api#charon-getdataset) unless there's a compelling reason. I am unsure if this fetch should _always_ be made, or there should be something about the dataset JSON which causes it to be made (note that tip-frequencies JSONs are only fetched if the dataset JSON specifies "frequencies" as one of the panels.)

### Context on why this is needed:
Auspice currently determines genotype coloring (e.g. https://nextstrain.org/zika?c=gt-NS1_349) via functions defined in https://github.com/nextstrain/auspice/blob/master/src/util/entropy.js which parse the mutations across a tree. If a position has no mutations, then auspice currently can't know what the base / aa is (e.g. https://nextstrain.org/zika?c=gt-NS1_350).

**Comment by jameshadfield** [comment]
This functionality was implemented in Auspice 2.19.0 via https://github.com/nextstrain/auspice/pull/1197
